packages/lodestar/src/api/impl/beacon/blocks/index.ts

```diff
@@ -135,6 +137,13 @@ export function getBeaconBlockApi({
       return {data: config.getForkTypes(block.message.slot).BeaconBlock.hashTreeRoot(block.message)};
     },
 
+    async publishBlindedBlock(signedBlindedBlock) {
+      const exeutionBuilder = chain.executionBuilder;
+      if (!exeutionBuilder) throw Error("exeutionBuilder required to publish SignedBlindedBeaconBlock");
+      const signedBlock = await exeutionBuilder.submitSignedBlindedBlock(signedBlindedBlock);
+      return await this.publishBlock(signedBlock);
```

Comment:
do we need to double check the returned payload is respective to the payload header we got in the blinded block in order not to spread mismatch state root block to the network?

also maybe we need some log here so that we can trace any issues if happen later?

Reply:
makes sense, can add the check in the execution/builder.ts itself for this to stay clean :+1:

Reply:
added in the builder